torch/_inductor/codegen/triton.py

```diff
@@ -1877,7 +1875,7 @@ def load(self, name: str, index: sympy.Expr):
             ep = ", eviction_policy='evict_last'"
         elif not is_coalesced:
             ep = ", eviction_policy='evict_last'"
-        elif self.inside_reduction and self.range_trees[-1].is_loop:
+        elif self.inside_reduction:
```

Comment:
This should probably apply to pointwise kernels as well?

Reply:
Oh right, I was planning to apply this to both pointwise and persistent reduction. Thanks for catching this!